.github/workflows/_linux_transformers.yml

```diff
@@ -106,12 +107,54 @@ jobs:
         id: installed
         run: |
           source activate huggingface_transformers_test
-          echo "TORCH_BRANCH_ID=$(python -c 'import torch; print(torch.__version__)')" |tee -a "${GITHUB_OUTPUT}" >> "${GITHUB_ENV}"
-          echo "TORCH_COMMIT_ID=$(python -c 'import torch; print(torch.version.git_version)')" |tee -a "${GITHUB_OUTPUT}" >> "${GITHUB_ENV}"
           echo "pip installed packages:"
           pip list | tee ${{ github.workspace }}/transformers/tests_log/pip_list.txt
+          echo "lspci gpu devices:"
+          lspci -d ::0380 | tee ${{ github.workspace }}/transformers/tests_log/lspci_0380.txt
```

Comment:
Can we use `xpu-smi discovery` to get the gpu devices information? Because the runner procession with `ZE_AFFINITY_MASK` by default, the `lspci` always show all devices information.

Reply:
Good idea. But let's add running xpu-smi in a stand-alone step. I remember it gives lengthy output by itself and for us it has special interest. Good to separate from this step. I will submit another PR on this since I need to experiment with xpu-smi a little for the better output.